image.go

```diff
@@ -20,6 +20,8 @@ type Image interface {
 	Target() ocispec.Descriptor
 	// Unpack unpacks the image's content into a snapshot
 	Unpack(context.Context, string) error
+	// Image provides access to internal image info
```

Comment:
@Random-Liu That is a much better solution. I am not quite sure about the role of this image object.